section3/exercises/loops-arrays.js

```diff
@@ -18,32 +18,54 @@ for (var i = 0; i < animals.length; i++) {
 
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
+let text = "";
 
+for (var i = 0; i < animals.length; i++) {
+  console.log(text = "The " + animals[i] + " is awesome!");
+}
 
 
 // YOU DO: Create an array of foods and then iterate over that array to log "Add
 // <food> to shopping list" for each food item.
+var foods = ["raddish", "potato", "pickles", "garlic"];
 
+for (var i = 0; i <foods.length; i++) {
+  console.log(text= "Add " + foods[i] + " to shopping list.");
+}
 
 
 // YOU DO: Create an array of numbers and then iterate over that array to log double
 // of each of the number. (Ex: 4 doubled is 8.)
+var numb = [1,3,5,7,12,22];
+
+for (var i = 0; i < numb.length; i++) {
+  console.log(numb[i] * 2);
+}
 
 
 // YOU DO: Using the array of names below, print out "Happy Birthday <name>" for
 // each name except the first two.
 var names = ["Sevy", "Lindsey", "Jericho", "Raeonna", "Latrina", "Jhana", "Luca"];
 
-
+for (var i = 2; i < names.length; i++) {
+  console.log(text = "Happy Birthday " + names[i]);
+}
 
 // THE FINAL YOU DO: Using the array of ages below, print out a statement for each age.
 // The age should determine which message is printed out:
   // Under 18? "Oh no, you can't vote quite yet."
   // 18 or over? "Make sure you get out and vote!"
 
 // WORD FROM THE WISE: Before you start coding, jot down, with pen and paper, what steps you
-// want to tell the program to take. Think of all the concepts you've learned so far. 
+// want to tell the program to take. Think of all the concepts you've learned so far.
 
 // Remember, stuck for 15-20 minutes? Reach out to your Mod 0 cohort on Slack.
 
-var ages = [17, 20, 34, 18, 16, 65, 49, 10, 22, 18];
\ No newline at end of file
+var ages = [17, 20, 34, 18, 16, 65, 49, 10, 22, 18];
+
+for (var i = 0; i < ages.length; i++) {
```

Comment:
Look up some examples in MDN docs of how conditional blocks (control flow, if/else blocks) should be formatted (indentation, curly braces).

More convention than function, but good convention will save you a ton of time spent debugging a missing curly brace at some point.